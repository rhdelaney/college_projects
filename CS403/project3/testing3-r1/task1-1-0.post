(((((lambda (x)
    (define y "v4")
    (define i -38)
    (define y -47)
    (lambda (x)
        (define y "v3")
        (define x 62)
        (define wcr 58)
        (define m 58)
        (lambda (x)
            (define y "v2")
            (lambda (x)
                (define y "v1")
                (staticScope 4 this)))))
 0) 1) 2) 3)
