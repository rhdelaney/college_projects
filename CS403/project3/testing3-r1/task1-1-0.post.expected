#[environment L0]
    y : v1
    x : 3
#[environment L1]
    y : v2
    x : 2
#[environment L2]
    m : 58
    wcr : 58
    x : 62
    y : v3
    x : 1
#[environment L3]
    y : -47
    i : -38
    y : v4
    x : 0
